source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -735,7 +736,15 @@ namespace
                 const PvtInput* input = attr->asA<PvtInput>();
                 if (input->isUniform())
                 {
-                    destPort = destNodeDef->addInput(attr->getName(), attr->getType().str());
+                    if (writeUniformsAsParameters)
```

Comment:
Write option to write uniforms as parameters. Previous change just made everything an input.